posts/2025-11-18-25.0.0.12-beta.adoc

```diff
@@ -0,0 +1,467 @@
+---
+layout: post
+title: "SpringBoot 4.0, preview of Jakarta Data 1.1 and more in 25.0.0.12-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/navaneethsnair1
+author_github: https://github.com/navaneethsnair1
+seo-title: "SpringBoot 4.0, preview of Jakarta Data 1.1 and more in 25.0.0.12-beta- OpenLiberty.io"
+seo-description: This beta release adds Spring Boot 4.x support, previews Jakarta Data 1.1 capabilities, and enhances the Netty-based HTTP transport. It also updates Model Context Protocol Server 1.0 and introduces the option to use your own AES-256 key for Liberty password encryption.
+blog_description: This beta release adds Spring Boot 4.x support, previews Jakarta Data 1.1 capabilities, and enhances the Netty-based HTTP transport. It also updates Model Context Protocol Server 1.0 and introduces the option to use your own AES-256 key for Liberty password encryption.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= SpringBoot 4.0, preview of Jakarta Data 1.1 and more in 25.0.0.12-beta
+Navaneeth S Nair <https://github.com/navaneethsnair1>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+This beta release adds Spring Boot 4.x support, previews Jakarta Data 1.1 capabilities, and enhances the Netty-based HTTP transport. It also updates Model Context Protocol Server 1.0 and introduces the option to use your own AES-256 key for Liberty password encryption.
+
+The link:{url-about}[Open Liberty] 25.0.0.12-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+
+* <<springboot, SpringBoot 4.0>>
+* <<data1.1, Early preview of Jakarta Data 1.1>>
```

Comment:
The link for this on the draft server doesn't go anywhere. Maybe it is confused by the `.` character?
[Early preview of Jakarta Data 1.1](https://blogs-draft-openlibertyio.mqj6zf7jocq.us-south.codeengine.appdomain.cloud/blog/2025/11/18/25.0.0.12-beta.html#data1.1)